docs/source/textdata.rst

```diff
@@ -154,6 +155,17 @@ It uses the following special chars:
     W         Insert a random uppercase word from the ipsum lorem word set. Always escaped
     ========  ======================================
 
+In all other cases, the char itself is used.
+
+The setting of the ``escapeSpecialChars`` determines how templates generate data.
+
+If set to False, then the template ``r"\\dr_\\v"`` will generate the values ``"dr_0"`` ... ``"dr_999"`` when applied
+to the values zero to 999. This conforms to earlier implementations for backwards compatibility.
+
+If set to True, then the template ``r"dr_\\v"`` will generate the values ``"dr_0"`` ... ``"dr_999"``
```

Comment:
Shouldnt this look different than when set to False?

Reply:
reworked this section